packages/web/src/components/TextField/_TextField.scss

```diff
@@ -0,0 +1,100 @@
+@use 'theme';
+
+.TextField {
```

Comment:
Figma says `Text-Input` so we should mirror that (without hyphens), is it right?

Reply:
We decided on Text Field in Figma and TextField in code. Figma is going to be updated soon.